Fixing Spacing in Admin Panel (Issue 381) + Comments

tallulahshepherd opened the pull request:
Fixed spacing in admin panel by adding margin-right parameter to buttons/checkboxes and margin-left parameter to the add meters message. This fixes #381 .

Previous documentation is included because I am unsure how to separate different commits from this pull request. 

Worked with @jamielepito

tallulahshepherd committed:
Merge pull request #1 from OpenEnergyDashboard/development

Updating personal fork

tallulahshepherd committed:
Tentative Comments

tallulahshepherd committed:
updated comments

tallulahshepherd committed:
Merge pull request #2 from OpenEnergyDashboard/development

Merging from OED proper to personal fork

tallulahshepherd committed:
Update LineChartContainer.ts

tallulahshepherd committed:
conflict fix

tallulahshepherd committed:
Merge branch OpenEnergyDashboard-development' into development jhgjdfhgjksss
i commit message

Commit Message

tallulahshepherd committed:
fix spacing in admin panel

tallulahshepherd committed:
Getting rid of outdated test code

huss commented:
This is on hold until the plotly PR #436 is merged. It also needs to have the travis issues fixed.

huss committed:
Merge branch 'development' of https://github.com/OpenEnergyDashboard/OED into pr/tallulahshepherd/431

huss committed:
fix small issues

- removes unused item causing lint issue
- adds spacing in PR to new language added later

huss committed:
simple lint fix

huss reviewed (approved):
All seems fine and it works after merging with development. I apologize to @tallulahshepherd  and @jamielepito for how long it took to accept this.
